packages/cfd/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -365,7 +365,7 @@ export default class CFDStore extends BaseStore {
                 this.account_type.type === 'dxtrade' || this.account_type.type === 'cTrader'
                     ? 'all'
                     : this.account_type.type,
-            company: '',
+            company: CFD_PLATFORMS.CTRADER ? values.company : '',
```

Comment:
We are already hardcoding the company in `createCFDAccount` since right now we only have 1 jurisdiction. companies are based on the `jurisdiction_selected_shortcode`.

Reply:
@hirad-deriv the issue was sth else, I've fixed it in the next commit.